### ISSUE TITLE
Could not get code signature for running application

### ISSUE THREAD

**Issue by dynamodeniz** [absalomedia/vagrant-manager#7]
**Describe the bug**
When I launch the app I get:

```
Uncaught Exception:
Error: Could not get code signature for running application
```

(followed by stack trace)

**To Reproduce**
1. Download app
2. Install app to Applications
3. Right click app and choose Open

**Expected behavior**
The app should launch.

**Screenshots**
[Error](https://i.imgur.com/9aT0CHq.png)

**Desktop (please complete the following information):**
 - OS: macOS Mojave 10.14.2
 - Browser: Irrelevant
 - Version: 2.1.3

**Additional context**
N/A

**Comment by absalomedia** [comment]
Related to #6

Unsure why this bug is still replicating. The code signing process shouldn't be firing any more.  Will do more testing over the next week.

**Comment by absalomedia** [comment]
Should be resolved by [2.1.4](https://github.com/absalomedia/vagrant-manager/releases/tag/2.1.4) release
